frameworks/solid/src/benchmark.jsx

```diff
@@ -219,22 +219,22 @@ const Benchmark = () => {
     console.log('starting benchmark');
     setTimeout(runBenchmark, 1000);
 
-    return (<Show when={data().length > 0}>
+    return (<Show when={data()}>
         <View ref={container}>
             <For each={ data() }>{ (row) => {
-                return <View x={row.x} y={row.y} width={row.width} height={row.height} color={row.color}>
-                    <Text 
+                return <node x={/*@once*/ row.x} y={/*@once*/ row.y} width={/*@once*/ row.width} height={/*@once*/ row.height} color={/*@once*/ row.color}>
```

Comment:
I'm no Solid expert but if I read this: https://docs.solidjs.com/reference/jsx-attributes/once
and quote:
```
Knowing this heuristic and its limitations, we can reduce overhead of things we know will never change by accessing them outside of the JSX.
```

Aside from the horrifying developer experience, wouldn't this break the tests where variables can no longer be updated?

Reply:
For extreme benchmarking it helps. For normal use cases you wouldn't need to do this. And those tests still work.

Reply:
That sets a bit of an unrealistic promise then doesn't it? I'm glad to see you're starting to see the overhead of JSX - however if this is not a realistic use case for using SolidJS + LightningJS and only exists for the sake of the benchmark then what's the value of the test?

What's next, forwarding the renderer createNode API and turn off everything for the love of the benchmark? IMHO the tests should be representative of how you would use it in an application.